### PR TITLE
Change Analytics Exporter jobs to run latest edx-platform code and config

### PR DESCRIPTION
**Pull request by pwnage101**
Also rename parameter CONFIG_FILENAME to EXPORTER_CONFIG_FILENAME to
avoid conflating it with the LMS/Studio config files.

DENG-369

Related changes:
* https://github.com/edx/edx-analytics-exporter/pull/49
* https://github.com/edx-ops/analytics-secure/pull/438
* https://github.com/edx-ops/baked-config-secure/pull/10

### CODE REVIEW

**Comment by pwnage101** [comment]
This draft still requires updates to the email optin version of the job.

**Review by brianhw — APPROVED** [review]
LGTM.  👍 

i assume the port to Python 3.7 is opportunistic, rather than required.

**Comment by pwnage101** [comment]
> i assume the port to Python 3.7 is opportunistic, rather than required.

Are you saying that edxapp in production is still running python 2?  Or <3.7?  FWIW, I think our jenkins only has 2.7 and 3.7, so there was no other choice.

**Comment by brianhw** [comment]
Yes, that makes sense -- of course it's required.   I thought it was just set up, but of course, that's the setup of the venv from which platform is run.

File: dataeng/jobs/analytics/AnalyticsExporter.groovy [hunk 1]
```diff
@@ -129,11 +133,12 @@ class AnalyticsExporter {
         }
 
         dslFactory.job('analytics-exporter-master') {
+            description('The Analytics Exporter weekly job, which exports tons of structure and state data for every course for every org and delivers them encrypted to our partners via S3.  Specifically, this sets up the shared edx-platform execution environment, fetches a list of all the orgs, then kicks off downstream analytics-exporter-worker jobs for each one.')
```
Comment (macdiesel) [inline dataeng/jobs/analytics/AnalyticsExporter.groovy:136]: Technically it only exports data for orgs who are at the partner level or above.

File: dataeng/jobs/analytics/AnalyticsExporter.groovy [hunk 2]
```diff
@@ -129,11 +133,12 @@ class AnalyticsExporter {
...
             parameters {
                 stringParam('ORGS', '*', 'Space separated list of organizations to process. Can use wildcards. e.g.: idbx HarvardX')
                 stringParam('EXPORTER_BRANCH', 'environment/production', 'Branch from the edx-analytics-exporter repository. For tags use tags/[tag-name].')
-                stringParam('PLATFORM_BRANCH', 'aed/analytics-exporter-settings-hotfix', 'Branch from the edx-platform repository. For tags use tags/[tag-name].')
-                stringParam('CONFIG_FILENAME', 'default.yaml', 'Name of configuration file in analytics-secure/analytics-exporter.')
+                stringParam('PLATFORM_BRANCH', 'tags/release-2020-08-20-14.08', 'Branch from the edx-platform repository. For tags use tags/[tag-name].')
```
Comment (macdiesel) [inline dataeng/jobs/analytics/AnalyticsExporter.groovy:140]: This branch tag does match the above which is tags/release-2020-09-08-16.55

**Review by macdiesel — CHANGES_REQUESTED** [review]
I noticed that the PLATFORM_BRANCH variable does not match. If intentional LMK and I can approve.

File: dataeng/jobs/analytics/AnalyticsExporter.groovy (hunk 2 above)
Comment (pwnage101) [reply dataeng/jobs/analytics/AnalyticsExporter.groovy:140]: good catch

**Review by macdiesel — APPROVED** [review]

**Pull request merged**